src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -267,8 +267,12 @@ private void buildElement(MessageMLParser context, org.w3c.dom.Element element)
     Element child = context.createElement(element, this);
     if (child != null) {
       child.buildAll(context, element);
-      child.validate();
-
+      try {
+        child.validate();
+      } catch (InvalidInputException e) {
+        context.clearBiContext();
```

Comment:
this is to prevent to have a partially created context? do we have other places where an error can happen?